libs/build_processed_dataset.py

```diff
@@ -46,6 +51,36 @@ def _get_abbrev_df():
     )
 
 
+def _get_testing_df():
+    # TODO: read this from a dataset class
+    abbrev_df = _get_abbrev_df()
+
+    ctd_df = CovidTrackingDataSource.local().data
+    ctd_df = ctd_df[CovidTrackingDataSource.TEST_FIELDS]
+    ctd_df = ctd_df.merge(
+            abbrev_df,
+            left_on='state',
+            right_on='abbreviation',
+            how="inner",
+            suffixes=['_dropcol', '']
+        ).drop("state_dropcol", axis=1)
+
+    return ctd_df
+
+
+def get_testing_timeseries_by_state(state):
+    test_df = _get_testing_df()
+    # use a string for dates
+    test_df['date'] = test_df.date.apply(lambda x: x.strftime("%m/%d/%y"))
+    test_df['positive'] = test_df['positive'].astype(int)
+    test_df['negative'] = test_df['negative'].astype(int)
+
+    state_test_df = test_df[test_df[CovidTrackingDataSource.Fields.STATE] == state]
+    # just select state
+    return state_test_df[['positive', 'negative', 'date']]
```

Comment:
Would love to refer to these by the class field names instead of the strings here